cdip_admin/integrations/management/commands/inreach_migration_script.py

```diff
@@ -0,0 +1,395 @@
+            try:
+                inreach_v2_auth_action = IntegrationAction.objects.get(
+                    value=inreach_auth_action_value,
+                    integration_type=inreach_v2_integration_type
+                )
+            except IntegrationAction.DoesNotExist:
+                self.stdout.write(
+                    f" -- Selected action {inreach_auth_action_value} does not exist in Gundi v2 for {inreach_integration_type} -- ")
+                return
+
+            # Get PUSH v2 action
+            inreach_push_action_value = "push_messages"
+            try:
+                inreach_v2_push_action = IntegrationAction.objects.get(
+                    value=inreach_push_action_value,
+                    integration_type=inreach_v2_integration_type
+                )
+            except IntegrationAction.DoesNotExist:
+                self.stdout.write(
+                    f" -- Selected action {inreach_push_action_value} does not exist in Gundi v2 for {inreach_integration_type} -- ")
+                return
+
+            inreach_er_integrations_created = 0
+            inreach_er_integrations_skipped = 0
+            inreach_er_integrations_with_error = 0
+            inreach_er_integration_configs_created = 0
+            inreach_er_destination_owners_created = 0
+            inreach_er_destination_integration_created = 0
+
+            # Step 2: ER to InReach
+            er_integration_type = "api_push" # API PUSH
+            try:
+                er_v2_integration_type = IntegrationType.objects.get(
+                    value=er_integration_type
+                )
+            except IntegrationType.DoesNotExist:
+                self.stdout.write(f" -- Selected integration type {er_integration_type} does not exist in Gundi v2 -- ")
+                return
+
+            # Get PULL v2 action
+            er_pull_action_value = "receive_data"
+            try:
+                er_v2_pull_action = IntegrationAction.objects.get(
+                    value=er_pull_action_value,
+                    integration_type=er_v2_integration_type
+                )
+            except IntegrationAction.DoesNotExist:
+                self.stdout.write(
+                    f" -- Selected action {er_pull_action_value} does not exist in Gundi v2 for {er_integration_type} -- ")
+                return
+
+            er_inreach_integrations_created = 0
+            er_inreach_integrations_skipped = 0
+            er_inreach_integrations_with_error = 0
+            er_inreach_integration_configs_created = 0
+            er_inreach_destination_integration_created = 0
+
+            self.stdout.write(f" -- Got {len(inbounds_to_migrate)} {inbound_type} inbounds to migrate -- \n\n")
+
+            for inbound in inbounds_to_migrate:
+                try:
+                    with transaction.atomic():
+                        inbound_owner, _ = Organization.objects.get_or_create(
+                            name=inbound.owner.name
```

Comment:
The owner should remain the same, so instead do this:

```suggestion
                        inbound_owner = inbound.owner
```

Reply:
I am doing this for the cases we read a v1 org that it is not registered in v2. 

Is that likely to happen? or I am too paranoid